### PR TITLE
[SPARKTA-520] Solr output template

### PR DESCRIPTION
**Pull request by danielcsant**
**Given** a Sparta user
**When** uses a Solr output in a policy
**Then** Sparta should write the aggregations on a Solr

### CODE REVIEW

**Comment by acemilio** [comment]
Please, check the documentation. I miss Solr link at the summary https://stratio.atlassian.net/wiki/display/SPARKTA/7.4+Outputs 
However, its documentation exists at the end of that page

**Comment by acemilio** [comment]
When yo are creating solr output on browser, the field **"Zookeeper Url"** should be **"Solr Url"**, shouldn't be?

**Comment by danielcsant** [comment]
The field is "Zookeeper Url". The output connects to the Zookeeper that Solr is using. I'll make it more clear in the documentation and add the missing links. Thanks!

**Comment by acemilio** [comment]
I can't store data into solr. Maybe I'm missing some step to run Solr.

- I run the command **/opt/solr/bin/solr start -z zk.demo.stratio.com:2181** in order to config Solr with my running zookeeper instance
- Setting the policy, I try to check/uncheck the property **"create schema"**

Finally, when the policy starts, the error below is thrown:
_2016-03-18 10:10:26 ERROR c.s.s.s.a.a.LocalSparkStreamingContextActor:63 - Error from server at http://localhost:8983/solr: Error CREATEing SolrCore 'cube1': Unable to create core [cube1] Caused by: Could not find configName for collection cube1 found:null_

**Comment by danielcsant** [comment]
By the moment it is not possible to persist the data in a Solr that is not in cloud mode due to the Solr connector we are using. To test it you can start Solr in cloud mode with `./bin/solr -e cloud`

Also I have mark in the documentation that it is mandatory to check the field `Is autocalculated id` when selecting the output. https://stratio.atlassian.net/wiki/display/SPARKTA/7.4+Outputs#id-7.4Outputs-8.4.11Solr

File: plugins/src/main/scala/com/stratio/sparta/plugin/output/solr/SolrDAO.scala [hunk 1]
```diff
@@ -151,7 +151,8 @@ trait SolrDAO extends Closeable with Logging {
     if (isCloud)
       new CloudSolrClient(zkHost)
     else
-      new HttpSolrClient("http://" + zkHost + "/solr")
+    // Not supported with current Spark-Solr connector
+      new HttpSolrClient("http://localhost:8983/solr")
```
Comment (anistal) [inline plugins/src/main/scala/com/stratio/sparta/plugin/output/solr/SolrDAO.scala]: Why localhost?

**Pull request closed without merging**